source/push-to-deploy.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Push To Deploy
+order: 19
+description: Learn how to setup Push to Deploy on Cloud, so you can deploy changes to your apps just buy by performing a commit to a repository.
+---
+
+With this feature, never was easier to create an app on Galaxy. All you need to do is follow three  simple steps, then you'll have a new app deployed.
+
+<h2 id="pre-requirements">Pre requirements</h2>
+
+Today we have our integration just with GitHub. In the future we'll add more Git providers.
+
+The only requirement you need today to start using this feature is to have a repository on GitHub with a Meteor project on it. And that's it.
+
+<h2 id="deploying-a-new-app">Deploying a new app</h2>
+
+Start by going to your Cloud account [here](https://cloud.meteor.com/). Then go to *apps* > *Deploy a new app*. Once there, you'll see this page:
```

Comment:
Do we have something in the UI called "Deploy a new app" ?

Reply:
Yes, even in the new design (app list) https://www.figma.com/file/m53xb9lWco3h5lUO3Y3K7J/Meteor-Cloud---Dashboard?node-id=1415%3A2482... but maybe we will change that...